### ISSUE TITLE
README: Should have cross-references between urdfdom_headers and urdfdom?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [ros/urdfdom_headers#63]
Both the READMEs in this repo and urdfdom do not reference each other at the top-level, which is confusing. I don't know when I should make an issue for `urdfdom_headers` or `urdfdom`, esp. related to the specification vs. implementation or whatever.

It would be nice if there were explicit guidance on that:
https://github.com/ros/urdfdom_headers/tree/2acf769db0f70205efd612830e2b2f2807862d80
https://github.com/ros/urdfdom/tree/0da4b20675cdbe14b532d484a1c17df85b4e1584

Relates SDFormat repo split as well:
https://github.com/osrf/sdf_tutorials/issues/2
https://github.com/osrf/sdf_tutorials/issues/10 (issue template)

**Comment by EricCousineau-TRI** [comment]
Ah, geez... Yet another repo?
https://github.com/ros/urdf/issues

**Comment by clalancette** [comment]
There's also https://github.com/ros/urdf_parser_py.git (for python) and https://github.com/ros/collada_urdf.git (which I believe is related, but I'd have to dig back into the tangled weave again to make sure).

I don't disagree that this is a mess, but it is unfortunately a difficult mess to fix.  Some of the packages are in Ubuntu packages, and some are released on the ROS buildfarm, which makes it worse.

**Comment by EricCousineau-TRI** [comment]
Ooh, yeah. I remember taking some time a couple years ago to at least indicate which forks are old. Seems like house-cleaning may be in order again :P

And I'm not calling for the repos to merged / futzed around with.
Rather, just for their explicit purposes to be called out (via `README.md` and an `issue_template.md`) so people can know when to look at one vs. the other (or realize that one of the repositories is effectively a meta-package).

**Comment by huangzonghao** [comment]
Yeah, can't agree more here. Took me sometime to figure out the relation between urdfdom and urdf

**Comment by clalancette** [comment]
For what it is worth, @IanTheEngineer recently updated the documentation to more clearly show the relationship.  It's available here: http://wiki.ros.org/urdf
